library/src/interactor_impl.cxx

```diff
@@ -817,7 +832,16 @@ bool interactor_impl::triggerCommand(std::string_view command)
     return true;
   }
 
-  const std::string& action = tokens[0];
+  std::string action = tokens[0];
+
+  // Resolve Alias
+  auto aliasIt = aliasMap.find(action);
+  if (aliasIt != aliasMap.end())
+  {
+    action = aliasIt->second;
```

Comment:
I dont get how can this work. `action` is supposed to be an action, not a full command.